indigo_app/static/javascript/indigo/views/annotations.js

```diff
@@ -543,13 +544,19 @@
     nextAnnotation: function(e) {
       e.preventDefault();
       e.stopPropagation();
-      this.gutter.activateNextItem().then((activeItem) => this.handleScrollForActiveItem(activeItem));
+      this.removeNewButton();
```

Comment:
This seems a bit odd to me. When the selection clears (because the user clicks elsewhere, etc.) then the button should remove itself.

Reply:
Clicking the prev/next button isn't particularly special and shouldn't have to do this, AFAIK.